### DISCUSSION TITLE
french cards

### DISCUSSION THREAD

**Discussion by leresteux** [backface/turtlestitch#127] [Q&A]
hi,
where can i find french beginner cards?

and if this doesn't exist, where can i find the layout for writing a translation ? 

THX

**Comment by chmod4711** [comment]
Not aware of French Beginner Cards, but will do research in the community!
In general, the English instruction cards are editable: https://www.turtlestitch.org/static/download/TurtleStitch-Cards-Beginners.pdf
Thanks for the question and of course it would be great if we could offer French instruction cards! Your help would be appreciated!

**Reply by backface** [reply]
TurtleStitch itself does not yet have full French translation! 

If you would want to get into that, that would be great, just take the German translation file as a template and replace it with French:
https://github.com/backface/turtlestitch/blob/master/stitchcode/locales/lang-de.js
https://github.com/backface/turtlestitch/blob/master/stitchcode/locales/lang-fr.js

Note that special accented characters need to be encoded - more info also here - but worst case I can do that easily once it is translated..
https://github.com/backface/turtlestitch/blob/master/locale/lang-fr.js

**Reply by leresteux** [reply]
hi
i did it and tested ; it's ok 

i am going in vacancies friday but i will try to ask to some friends to read my translation ... maybe i can make better

**Comment by leresteux** [comment]
hi
 i am not a good user of github
so i hope you have got my "pull"
or you can find the translation here : https://github.com/leresteux/turtlestitch/blob/master/stitchcode/locales/lang-fr.js

**Comment by backface** [comment]
I did not get the pull but I copied the file. It's integrated. Thanks a lot!

**Reply by leresteux** [reply]
U are welcome.

In september a will check it with friend to see if it s possible to have a better translation.

I will back to you.

Good summer.
